### PR TITLE
added profiling docs

### PR DESCRIPTION
**Pull request by enekofb**
Part of https://github.com/weaveworks/weave-gitops-enterprise/issues/3228 

**What changed?**
- added profiling docs to monitoring
- updated monitoring docs to match changes in https://github.com/weaveworks/weave-gitops-enterprise/issues/3228

**How did you validate the change?**

Developed locally

Screenshot 2023-10-18 at 12 01 51

### CODE REVIEW

File: website/docs/operations/monitoring.mdx [hunk 1]
```diff
@@ -130,3 +139,20 @@ Monitor Weave Gitops GO runtime metrics like Memory Usage, Memory Heap, Goroutin
 **Explorer**
 
 Monitor Explorer golden signals. More info [here](../../explorer/operations#monitoring)
+
+### Profiling
+
+Profiling can be useful during operation to help you to gain deeper understanding of the management cluster is behaving a runtime.
```
Comment (opudrovs) [inline website/docs/operations/monitoring.mdx:145]: `of the management cluster is behaving a runtime` should it be `of how the management cluster is behaving a runtime`?

`Go,profiling` - a space before the comma is missing.

File: website/docs/operations/monitoring.mdx [hunk 2]
```diff
@@ -130,3 +139,20 @@ Monitor Weave Gitops GO runtime metrics like Memory Usage, Memory Heap, Goroutin
...
+Given Weave GitOps is writen in Go,profiling happens using [pprof](https://pkg.go.dev/runtime/pprof) that Weave GitOps
+exposes via [pprof http handler](https://pkg.go.dev/net/http/pprof)
+
+
+#### Get Started
+
+1. You have [enabled](#setup) the monitoring server with the profiling endpoint.
+2. Navigate to your monitoring server url to the `/debug/pprof` path where the pprof web interface is exposed.
```
Comment (opudrovs) [inline website/docs/operations/monitoring.mdx:153]: Should `url` be uppercase?

**Review by opudrovs — APPROVED** [review]
LGTM, only minor comments.

File: website/docs/operations/monitoring.mdx (hunk 1 above)
Comment (enekofb) [reply website/docs/operations/monitoring.mdx]: thanks, updated to make it a bit more clear.

**Pull request merged**